### PR TITLE
updating docs to refer to actions from "main" branch

### PR DESCRIPTION
**Pull request by dlockhart**
I've created a `main` branch off `master` and this PR just updates the documentation to refer to actions from main instead of master.

After this I'll switch the default branch to be `main`, which will leave `master` as a placeholder until we make some breaking changes and update everyone to point to main.

### CODE REVIEW

File: prepare-translations-for-npm/README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ jobs:
         uses: Brightspace/third-party-actions@actions/setup-node
         # additional validation steps can be run here
       - name: Prepare Translations for NPM
-        uses: BrightspaceUI/actions/prepare-translations-for-npm@master
+        uses: BrightspaceUI/actions/prepare-translations-for-npm@main
         with:
           LANG_PATH: lang
         # NPM publishing step happens here
```
Comment (svanherk) [inline prepare-translations-for-npm/README.md:35]: Could update the link to the localize mixin below to use `main` while we're here

File: semantic-release/README.md [hunk 2]
```diff
@@ -33,8 +33,9 @@ jobs:
       - name: Setup Node
         uses: Brightspace/third-party-actions@actions/setup-node
       - name: Semantic Release
-        uses: BrightspaceUI/actions/semantic-release@master
+        uses: BrightspaceUI/actions/semantic-release@main
         with:
+          DEFAULT_BRANCH: main
```
Comment (svanherk) [inline semantic-release/README.md:38]: We should update `semantic-release`'s default to be `main` for this version, and then when we move everyone over we can switch them

Comment (dlockhart) [reply semantic-release/README.md:38]: That's a great idea. So would we rename their branches for them, or just add in `DEFAULT_BRANCH: master`?

Comment (svanherk) [reply semantic-release/README.md:38]: I was thinking just add `DEFAULT_BRANCH: master` until they have time to get to it, to be less disruptive

Comment (dlockhart) [reply semantic-release/README.md]: Done.

**Review by svanherk — APPROVED** [review]

**Pull request merged**